app/c2/gist.py

```diff
@@ -0,0 +1,144 @@
+import json
+import uuid
+import aiohttp
+import re
+from base64 import b64encode
+
+from app.objects.c_c2 import C2
+
+
+class Gist(C2):
+
+    def __init__(self, services, module, config, name):
+        self.key = config.get('key')
+        super().__init__(services, module, config, name)
+
+    def get_key(self):
+        """
+        Returns this C2 objects api key
+        :return: GIST api key
+        """
+        return self.key
```

Comment:
get config would be better

Reply:
If we change this, we need to update the sandcat compilation as well @brianedmonds90